File: escobar/application/controllers/Contatos.php
<?php

class Contatos extends CI_Controller
{

    public function index()
    {

        $this->load->model("Contatos_Model");


        $contatos = $this->Contatos_Model->buscarTodos();

        $pacote = array(
            "contatos" => $contatos,
            "pagina" => "tabela.php"
        );

        $this->load->view("index", $pacote);
    }




    //Controller Criar contato
    public function novo()
    {
        $pacote = array(
            "pagina" => "contatoNovo.php"
        );
        $this->load->view("index", $pacote);
    }
    //Controller Salvar Novo
    public function salvarNovo()
    {
        $dados = array(
            'nome' => $_POST['nome'],
            'telefone' => $_POST['telefone'],
            'email' => $_POST['e-mail'],
            'cidade' => $_POST['cidade'],
            'estado' => $_POST['estado'],
            'rua' => $_POST['rua'],
            'cep' => $_POST['cep'],
            'bairro' => $_POST['bairro'],
            'pais' =>$_POST['pais']
        );
        $this->load->model("Contatos_Model");
        $this->Contatos_Model->salvarNew($dados);


        redirect('contatos');
    }
    //Controller Alterar
    public function alterar($identificador)
    {
        $this->load->model("Contatos_Model");

        $contato = $this->Contatos_Model->buscarId($identificador);
        $pacote = array(
            "contato" => $contato,
            "pagina" => "contatoAlterar.php"
        );
        $this->load->view('index', $pacote);
    }
    //Controller Salvar
    public function salvarAlterar(){
        $id = $_POST['id'];
        $nome = $_POST['nome'];
        $telefone = $_POST['telefone'];
        $email = $_POST['email'];
        $rua = $_POST['rua'];
        $bairro = $_POST['bairro'];
        $cep = $_POST['cep'];
        $cidade = $_POST['cidade'];
        $estado = $_POST['estado'];
        $pais = $_POST['pais'];

        $this->load->model("Contatos_Model");
        $this->Contatos_Model->salvarAlterar($id, $nome, $telefone, $email, $rua, $bairro, $cep, $cidade, $estado, $pais );
        redirect("contatos");
    }
    //Controller Excluir
    public function apagarContato($id){
        $this->load->model("Contatos_Model");
        $this->Contatos_Model->apagarContato($id);
        redirect("contatos");
        
    }
    public function endereco()
    {

        $this->load->model("Contatos_Model");


        $tbendereco = $this->Contatos_Model->endereco();

        $pacote = array(
            "contatos" => $tbendereco,
            "pagina" => "tabelaEnd.php"
        );

        $this->load->view("index", $pacote);
    }

}

File: contatos.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Text;
namespace Agenda
{
    #region Contatos
    public class Contatos
    {
        #region Member Variables
        protected int _id;
        protected string _nome;
        protected string _telefone;
        protected string _email;
        protected int _validar;
        protected string _pais;
        protected string _cidade;
        protected string _estado;
        protected string _rua;
        protected string _cep;
        protected string _bairro;
        #endregion
        #region Constructors
        public Contatos() { }
        public Contatos(string nome, string telefone, string email, int validar, string pais, string cidade, string estado, string rua, string cep, string bairro)
        {
            this._nome=nome;
            this._telefone=telefone;
            this._email=email;
            this._validar=validar;
            this._pais=pais;
            this._cidade=cidade;
            this._estado=estado;
            this._rua=rua;
            this._cep=cep;
            this._bairro=bairro;
        }
        #endregion
        #region Public Properties
        public virtual int Id
        {
            get {return _id;}
            set {_id=value;}
        }
        public virtual string Nome
        {
            get {return _nome;}
            set {_nome=value;}
        }
        public virtual string Telefone
        {
            get {return _telefone;}
            set {_telefone=value;}
        }
        public virtual string Email
        {
            get {return _email;}
            set {_email=value;}
        }
        public virtual int Validar
        {
            get {return _validar;}
            set {_validar=value;}
        }
        public virtual string Pais
        {
            get {return _pais;}
            set {_pais=value;}
        }
        public virtual string Cidade
        {
            get {return _cidade;}
            set {_cidade=value;}
        }
        public virtual string Estado
        {
            get {return _estado;}
            set {_estado=value;}
        }
        public virtual string Rua
        {
            get {return _rua;}
            set {_rua=value;}
        }
        public virtual string Cep
        {
            get {return _cep;}
            set {_cep=value;}
        }
        public virtual string Bairro
        {
            get {return _bairro;}
            set {_bairro=value;}
        }
        #endregion
    }
    #endregion
}